src/types/annotation.ts

```diff
@@ -12,6 +12,13 @@ export interface AnnotationStream {
   annotations: Annotation[]
 }
 
+export interface AnnotationStreamDetail {
```

Comment:
I think we should call this `AnnotationStream` and we should rename the current `AnnotationStream` to `AnnotationResponse`